phpBB/includes/functions.php

```diff
@@ -3579,6 +3579,8 @@ function phpbb_quoteattr($data, $entities = null)
 /**
 * Get user avatar
 *
+* @deprecated 4.0.0 Use \phpbb\avatar\helper::get_user_avatar() instead
```

Comment:
Move this functions to functions_compatibility.php if you can

Reply:
Okay, so simply move them to there? Will do.